awx/main/tests/functional/api/test_inventory.py

```diff
@@ -60,6 +60,44 @@ def test_inventory_source_unique_together_with_inv(inventory_factory):
     is2.validate_unique()
 
 
+@pytest.mark.django_db
+def test_inventory_host_name_unique(scm_inventory, post, admin_user):
+    inv_src = scm_inventory.inventory_sources.first()
+    group = inv_src.groups.create(name='barfoo', inventory=scm_inventory)
+    group.save()
```

Comment:
there's no need to call the `.save()` after a `.create`. You should be able to delete this line